### DISCUSSION TITLE
JavaScript Debug Terminal does not work

### DISCUSSION THREAD

**Discussion by gdrdrdr** [microsoft/vscode-discussions#706] [Extension Development QnA]
JavaScript Debug Terminal was working as expected. i dont know what has changed on my Macbook and it stopped working. Now my JavaScript Debug Terminal acts just like a normal terminal. It does not stop at breakpoints nor show call stack and stuff. 

this is the command i run. i use typescript to run cumcumber BDD tests
`./node_modules/.bin/protractor dist/protractor_conf.js --cucumberOpts.tags="@demo1"`
![image](https://github.com/microsoft/vscode-discussions/assets/84905160/16d27bc8-f91e-44eb-9fa2-3677cc4d2127)

another change i noticed is that when i do tsc: watch, it gives call stack which was not the case before
![image](https://github.com/microsoft/vscode-discussions/assets/84905160/99631300-7558-4d88-a9a8-64af8b6fc5c5)
when click "Disconnect":
it gives:
`
Debugger listening on ws://127.0.0.1:65442/5b5d9bec-2828-47f8-bf2e-5a67c2e0ee40
For help, see: https://nodejs.org/en/docs/inspector
`
![image](https://github.com/microsoft/vscode-discussions/assets/84905160/2d797298-b862-4e5c-90f1-1bf74410c2a4)
![image](https://github.com/microsoft/vscode-discussions/assets/84905160/4c98dfd2-d29d-4d6e-9fd4-3a5ce66c711b)

maybe this has something to do with the debugger issue?

**Comment by olguzzar** [comment]
Hi, this forum is specifically for developing extensions. If you have a general VS Code issue, please file it on the [main VS Code repo](https://github.com/microsoft/vscode/issues). Thanks!
